app/models/client_contacts.rb

```diff
@@ -0,0 +1,239 @@
+class ClientContacts
+  # this class represents the modal for editing all of a client's
+  # various contact categories together
+
+  # Lots of duplication because of the different associations managed.
+
+  include ActiveModel::Model
+  include UpdateableAttributes
+
+  attr_accessor :client,
+                :shelter_agency_contact_ids,
+                :regular_contact_ids,
+                :dnd_staff_contact_ids,
+                :housing_subsidy_admin_contact_ids,
+                :ssp_contact_ids,
+                :hsp_contact_ids,
+                :do_contact_ids
+
+  delegate :to_param, to: :client
```

Comment:
This seems kind of dangerous... why are using the client id for the client contact id?

Reply:
Ahh.. i see it's a presenter of some kind not backed by a real table. Carry-on